components/p3a/metric_names.h

```diff
@@ -177,14 +177,22 @@ constexpr inline auto kCollectedSlowHistograms =
   base::MakeFixedFlatSet<base::StringPiece>({
     // Please remove following placeholder metric once
     // static slow metrics are added
-    "Brave.P3A.ExampleSlowMetric"
+    "Brave.P3A.TestSlowMetric",
 });
 
 constexpr inline auto kCollectedExpressHistograms =
   base::MakeFixedFlatSet<base::StringPiece>({
-    // Please remove following placeholder metric once
-    // static express metrics are added
-    "Brave.P3A.ExampleExpressMetric"
+    // Keep the top metric for the express unit test
+    // Remove it when there are other non ephemeral metrics at the beginning of the set
+    "Brave.P3A.TestExpressMetric",
+    "Brave.Rewards.EnabledInstallationTime"
+});
+
+// List of metrics that should only be sent once per latest histogram update.
+// Once the metric value has been sent, the value will be removed from the log store.
+constexpr inline auto kEphemeralHistograms =
```

Comment:
we need to start spec'ing all these details, b/c new concepts like ephemeral metrics are not really clear

Reply:
yes, 100% agree. i have a comprehensive & technical P3A document in the works